core/string/ustring.h

```diff
@@ -186,7 +186,9 @@ class [[nodiscard]] CharStringT {
 	_FORCE_INLINE_ operator Span<T>() const { return Span(ptr(), length()); }
 	_FORCE_INLINE_ Span<T> span() const { return Span(ptr(), length()); }
 
-	_FORCE_INLINE_ Error resize(int p_size) { return _cowdata.template resize<false>(p_size); }
+	/// Resizes the string. The given size must include the null terminator.
+	/// New characters are not initialized, and should be set by the caller.
+	_FORCE_INLINE_ Error resize_uninitialized(int64_t p_size) { return _cowdata.template resize<false>(p_size); }
```

Comment:
You changed this to take in an int64_t, but `size()` still returns an int. Same with `length()`.

Reply:
Yes, that's something I want to address one of these days :)

Reply:
It shouldn't be, Gdextension bound methods are using int64_t anyway, and scripting likewise. Modules have to be recompiled (as always).